test/unit/specs/composables/useBadgeData.spec.js

```diff
@@ -502,4 +502,52 @@ describe('useBadgeData.js', () => {
 			expect(getHighestPriorityDisplayBadge(badges)).toEqual({ id: ID_CLIMATE_ACTION, level: 3 });
 		});
 	});
+
+	describe('getCompletedBadges', () => {
+		it('should return the completed badges', () => {
```

Comment:
Would probably be good to add a test or two for bad states, like when badges not defined.